### DISCUSSION TITLE
Compatible with Chat GPT?

### DISCUSSION THREAD

**Discussion by vijishvanya** [tectalichq/public-openai-client-php#14] [General] [answered]
Hello,
Is this compatible with gpt-3.5-turbo that opened up recently?
gpt-3.5-turbo uses "messages" instead of "prompts".
Thanks

**Comment by thejamescollins** [comment, marked as answer]
Hi there @vijishvanya,

I'm pleased to advise that [version 1.4.0 has just been released](https://github.com/tectalichq/public-openai-client-php/releases/tag/v1.4.0). This new release adds support for ChatGTP and the `gpt-3.5-turbo` model, as well as audio transcription and translation.

Please see our [the examples](https://tectalic.com/apis/openai/docs/php#examples) in our documentation for details on how to make use of the new `ChatCompletions` functionality.

All the best,

James

**Reply by Wellsj27** [reply]
I am having an issue following the example, I am not sure what to use or import at the top because chatCompletion()-> create is undefined 
<img width="1440" alt="Screenshot 2023-03-16 at 12 54 16 PM" src="https://user-images.githubusercontent.com/58715629/225724376-d5b6b912-af7d-46be-a3fd-88c2171d1e74.png">

**Reply by thejamescollins** [reply]
Hi @Wellsj27,

That `chatCompletions()` method (and associated models) was added in v1.4.0 of `tectalic/openai`.

Is the error still occurring after you ran that `composer update` command? I see that you've updated in your screenshot, but does the error still occur?

If you copy the [Text Completion using ChatGPT](https://github.com/tectalichq/public-openai-client-php#text-completion-using-chatgpt) example, does that work for you?

Please send back the full error message so that we can assist in more detail.

Thanks,

James

**Reply by Wellsj27** [reply]
Everything is updated and the error still occurs this is the error I am getting. 
<img width="1440" alt="Screenshot 2023-03-17 at 9 22 51 AM" src="https://user-images.githubusercontent.com/58715629/225949765-43c59285-8ee2-4e9e-aff8-03476893bf96.png">

**Reply by thejamescollins** [reply]
> Everything is updated and the error still occurs this is the error I am getting. 

Does an error occur when the code runs, or is it just an Intelliphence warning/issue?

**Comment by vijishvanya** [comment]
Hello,
Would this be compatible with GPT4 too? I hope to get the GPT access soon? Just wanted to understand what changes I may need to do in my code via tectalic API? Thanks much.

**Reply by thejamescollins** [reply]
Hi @vijishvanya,

Yes it is, please see the readme for details: https://github.com/tectalichq/public-openai-client-php

If you are already using the chatCompletions endpoint with GPT-3.5, the only change needed is to change the ‘model’ request parameter. 

James
